### PR TITLE
Add scatter tool test

### PR DESCRIPTION
**Pull request by wh1210**
* Add scatter tool test
* Move place explorer on top of the screenshot part.

### CODE REVIEW

File: server/webdriver_tests/scatter_test.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        expanded_menu.find_element_by_id("checkbox-Person,count").click()
+        expanded_menu.find_element_by_id("checkbox-Person,age").click()
+
+        # Choose parameters.
+        selects = Select(self.driver.find_element_by_id("place-types"))
+        selects.select_by_value("CensusCoreBasedStatisticalArea")
+        time.sleep(3)
+
+        # Assert there is a chart showing up.
+        chart = self.driver.find_element_by_id("chart-div").find_elements_by_tag_name("svg")
```
Comment (shifucun) [inline server/webdriver_tests/scatter_test.py:54]: nit: can you format this by indent a new line from "chart-div"

File: server/webdriver_tests/scatter_test.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        self.assertEqual(len(chart), 1)
```
Comment (shifucun) [inline server/webdriver_tests/scatter_test.py:55]: can you try to assert deeper element in the svg

File: server/webdriver_tests/scatter_test.py (hunk 1 above)
Comment (wh1210) [reply server/webdriver_tests/scatter_test.py]: Done

File: server/webdriver_tests/scatter_test.py (hunk 2 above)
Comment (wh1210) [reply server/webdriver_tests/scatter_test.py:56]: The svg component is a little messy. I was trying to find some valuable things, and ending with assert there are many circles showing up in the chart.

Comment (shifucun) [reply server/webdriver_tests/scatter_test.py:56]: check circle count is good

**Pull request merged**